.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -17,6 +17,7 @@ body:
         - The core library
         - discord.ext.commands
         - discord.ext.tasks
+        - discord.ext.menus
```

Comment:
might wanna put this on hold since #589 renames it to ext.pages

Reply:
You can rename it in your pr